recipes/napari-tomoslice/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "napari-tomoslice" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-tomoslice-{{ version }}.tar.gz
+  sha256: 3c8bc659df18e4eaebb975a3a85714ee087c78569fc1bc3bc3c44e399f878bcc
+
+build:
+  entry_points:
+    - napari-tomoslice = napari_tomoslice.cli:cli
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - mrcfile
+    - napari ==0.4.12
```

Comment:
As in the other recipe, is this the only version known to work or can we relax the restrains a bit? @alisterburt

Reply:
cc here, this is the only known working version, I can try to update soon but no guarantees on timescale